modeltrans/forms.py

```diff
@@ -18,6 +18,10 @@ def __init__(self, options=None):
         super().__init__(options)
         self.languages = getattr(options, "languages", ["browser", "fallback"])
         self.fallback_language = getattr(options, "fallback_language", None)
+        self.labels = {
+            "fallback": getattr(options, "fallback_label", _("fallback language")),
+            "translation": getattr(options, "translation_label", _("translation language")),
+        }
```

Comment:
In the more compact version, this would become
```suggestion
        labels = getattr(options, "labels", {})
        self.labels = {
            "fallback": labels.get("fallback_label", _("fallback language")),
            "translation": labels.get("translation_label", _("translation language")),
        }
```